yearn/prices/uniswap/v3.py

```diff
@@ -63,7 +63,7 @@ def get_price(self, token, block=None):
 
         paths += [[token, fee, usdc] for fee in self.fee_tiers]
 
-        scale = 10 ** contract(token).decimals()
+        scale = 10 ** interface.ERC20(token).decimals()
```

Comment:
avoid interfaces before this is fixed https://github.com/eth-brownie/brownie/issues/1092